recipes/scikit-digital-health/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "scikit-digital-health" %}
+{% set version = "0.9.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PfizerRD/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 8adc61af12e92e047a06f276a32a56cdf4deca350f8bb24df95c25ea177fc749
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+  host:
+    - python
+    - numpy
+    - libzip
+    - gsl
```

Comment:
Doesn't pip need to be included here? @ocefpaf

Reply:
Argh. Missed that one. Yes, it does, I even said so above but got lost in the changes. (This is what I get for reviewing on mobile during a GH funny outage.)

Reply:
>  GH funny outage.

Yeah I was trying to comment but nothing went through – seems to be fixed now though. 

I'll make a PR in the feedstock. 

Hats off though for reviewing on mobile. I can barely make out the meta.yaml there. 😄

Reply:
https://github.com/conda-forge/scikit-digital-health-feedstock/pull/1